src/R/Components/Impl/ConnectionManager/IConnection.cs

```diff
@@ -2,15 +2,14 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
-using System.Threading.Tasks;
 
 namespace Microsoft.R.Components.ConnectionManager {
-    public interface IConnection {
+    public interface IConnection: IConnectionInfo {
```

Comment:
To reuse implementation `ConnectionInfo` and also `IConnection` looks like live connection while `info`  is it's persistent part. Was there a reason they were separate?